cashapppay/src/main/res/template/values/strings.xml.tt

```diff
@@ -8,5 +8,4 @@
 
 <resources>
     <string name="cash_app_pay_store_payment_method">%%storeDetails%%</string>
-    <string name="cash_app_pay_waiting_text">%%paypal.processingPayment%%</string>
```

Comment:
This is technically a breaking change, if anyone is overriding the string. How necessary is it?

Reply:
We decided it's okay as there is minimal impact. Added a entry in the release notes as well